sherpa-onnx/csrc/funasr-nano-tokenizer.h

```diff
@@ -0,0 +1,108 @@
+// sherpa-onnx/csrc/funasr-nano-tokenizer.h
+//
+// A self-contained Qwen3 ByteLevel-BPE tokenizer implementation.
+// - No dependency on tokenizers-cpp / HF tokenizers
+// - Loads vocab.json + merges.txt + tokenizer.json(added_tokens)
+// - Supports AddedTokens via Trie longest-match
+// - ByteLevel bytes_to_unicode encode/decode
+// Author: zengyongwang
```

Comment:
Was this file authored by you? Additionally, could you add copyright information to the other files included in this pull request?